openpower/scripts/release-notes

```diff
@@ -0,0 +1,152 @@
+#!/bin/env perl
+use strict;
+
+my $repos =
+{
+    'op-build' => { REPO => "http://github.com/open-power/op-build" },
+    'hostboot' => { REPO => "http://github.com/open-power/hostboot" ,
+                    DIR => "openpower/package/hostboot",
+                    PACKAGE => "HOSTBOOT" },
+    'skiboot'  => { REPO => "http://github.com/open-power/skiboot" ,
+                    DIR => "openpower/package/skiboot",
+                    PACKAGE => "SKIBOOT" },
+    'occ'      => { REPO => "http://github.com/open-power/occ" ,
+                    DIR => "openpower/package/occ",
+                    PACKAGE => "OCC" },
+    'pnor'     => { REPO => "http://github.com/open-power/pnor" ,
+                    DIR => "openpower/package/openpower-pnor",
+                    PACKAGE => "OPENPOWER_PNOR" },
+    'palmetto-xml' => { REPO => "http://github.com/open-power/palmetto-xml" ,
+                    DIR => "openpower/package/palmetto-xml",
+                    PACKAGE => "PALMETTO_XML" },
+    'habanero-xml' => { REPO => "http://github.com/open-power/habanero-xml" ,
+                    DIR => "openpower/package/habanero-xml",
+                    PACKAGE => "HABANERO_XML" },
+    'firestone-xml' => { REPO => "http://github.com/open-power/firestone-xml" ,
+                    DIR => "openpower/package/habanero-xml",
+                    PACKAGE => "FIRESTONE_XML" },
+};
+
+my $begin_release = shift;
+my $end_release = shift;
+
+foreach my $repo (keys %{$repos})
+{
+    if (-e $repo)
+    {
+        system("cd $repo; git fetch") && die "Could not fetch $repo";
+    }
+    else
+    {
+        system("git clone $repos->{$repo}->{REPO} $repo") &&
+            die "Could not clone $repo";
+    }
+}
+
+system("cd op-build; git checkout $end_release --force; git reset HEAD --hard");
+
+open(OP_SUMMARY, "cd op-build; git diff $begin_release...$end_release ".
```

Comment:
Is git diff supposed to have 3 '.' between $begin_release and $end_release, assuming this works since things look good.